workspaces/orchestrator/plugins/scaffolder-backend-module-orchestrator/src/actions/runWorkflow.ts

```diff
@@ -53,22 +54,46 @@ export const createRunWorkflowAction = (
     supportsDryRun: true,
     schema: {
       input: {
-        required: ['parameters'],
+        required: ['workflow_id', 'targetEntity', 'parameters'],
         type: 'object',
         properties: {
+          workflow_id: {
+            type: 'string',
+            title: 'Workflow ID',
+            description:
+              'The workflow identifier from the workflow definition.',
+          },
+          targetEntity: {
```

Comment:
since workflow_id is snake case, change targetEntity to target_entity for consistency